fetch-publication/src/test/java/no/unit/nva/publication/fetch/FetchPublicationHandlerTest.java

```diff
@@ -75,52 +78,58 @@ public void setUp() {
     @DisplayName("handler Returns Ok Response On Valid Input")
     public void handlerReturnsOkResponseOnValidInput() throws IOException, ApiGatewayException {
         Publication publication = createPublication();
-        when(publicationService.getPublication(any(UUID.class)))
-            .thenReturn(publication);
+        when(publicationService.getPublication(any(UUID.class))).thenReturn(publication);
 
-        fetchPublicationHandler.handleRequest(inputStream(), output, context);
+        fetchPublicationHandler.handleRequest(generateHandlerRequest(), output, context);
 
-        GatewayResponse gatewayResponse = objectMapper.readValue(output.toString(), GatewayResponse.class);
+        GatewayResponse<PublicationResponse> gatewayResponse = objectMapper.readValue(output.toString(),
```

Comment:
refactor the command `objectMapper.readValue(...)` to a private  `parseHandlerResponse()` method.